packages/frameworks/src/frameworks.ts

```diff
@@ -1618,6 +1618,58 @@ export const frameworks = [
     buildCommand: 'vite build',
     getOutputDirName: async () => 'dist',
   },
+  {
+    name: 'Astro',
+    slug: 'astro',
+    demo: 'https://astro.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/astro.svg',
+    tagline:
+      'Astro is a next-generation static site generator that mixes lightning-fast performance with modern developer experience.',
+    description: 'An Astro site, using the default starter kit.',
+    website: 'https://astro.build',
+    envPrefix: 'SNOWPACK_PUBLIC_',
```

Comment:
Should this be `ASTRO_`?

Reply:
I believe this is still `SNOWPACK_PUBLIC_` since internally it is using `snowpack`. I don't believe this has been aliased yet. @fredkschott can you confirm?

Reply:
Sorry, I should have been more clear. This is used for Vercel.

https://vercel.com/docs/environment-variables#system-environment-variables

<img width="856" alt="CleanShot 2021-07-26 at 09 57 37@2x" src="https://user-images.githubusercontent.com/9113740/127028778-83cff9ac-c29d-45a0-b459-817f80344156.png">

Reply:
Sorry, I think I'm missing the point of this property. The comment on the Framework typing says it is used to inline values into the browser bundle.  

```ts
  /**
   * The environment variable prefix used to inline values into the browser bundle.
   * @example "NEXT_PUBLIC_"
   */
  envPrefix?: string;
```

This would be how the developer could utilize the env vars in their builds, correct? If I wanted to use `VERCEL_ENV` in my browser build I would need to use `NEXT_PUBLIC_VERCEL_ENV` for next? If so, I believe it would be `SNOWPACK_PUBLIC_VERCEL_ENV`?

Reply:
Yep, that sounds right 👍

Reply:
@styfle I don't believe there is anything in the docs currently. I know the team just launched the docs site recently. I will file an issue to add it to the docs. In the meantime, you can see that the docs site itself utilizes the `SNOWPACK_PUBLIC_` prefix. 

https://github.com/snowpackjs/astro/blob/main/docs/README.md
https://github.com/snowpackjs/astro/blob/main/docs/src/components/AvatarList.astro#L9

Reply:
Ok lets wait to see what happens with the upstream issue.

I'm also wondering if we should wait for JS files to be hashed so we can cache them per [this comment](https://github.com/vercel/vercel/pull/6477#discussion_r669937050)

Reply:
Sounds good! I will also open a RFC around additional file hashing in the production builds to keep the conversation going.